src/app/model/Mission.js

```diff
@@ -1,5 +1,12 @@
 import { get, filter } from "lodash";
 
+export const missionStatusDict = {
```

Comment:
Maybe throw this in mission model while you at it

Reply:
hi @mat10tng, not sure what you mean, can you please provide example?